jea/security-considerations.md

```diff
@@ -135,16 +163,21 @@ A more secure version of this same role capability would look like:
 }
 ```
 
-Avoid using wildcards in role capabilities, and be sure to [audit effective user permissions](audit-and-report.md#check-effective-rights-for-a-specific-user) regularly to understand which commands a user has access to.
+Avoid using wildcards in role capabilities. Be sure to regularly [audit effective user permissions](audit-and-report.md#check-effective-rights-for-a-specific-user)
+to understand which commands are accessible to the user.
 
-## JEA does not protect against admins
+## JEA doesn't protect against admins
 
-One of the core principles of JEA is that it allows non-admins to perform *some* admin tasks.
-JEA does not protect against those who already have administrator privileges.
-Users who belong "domain admins," "local admins," or other highly privileged groups in your environment will still be able to get around JEA's protections by signing into the machine via another means.
-They could, for example, sign in with RDP, use remote MMC consoles, or connect to unconstrained PowerShell endpoints.
-Local admins on the system can also modify JEA configurations to allow additional users to manage the system or change a role capability to extend the scope of what a user can do in their JEA session.
-It is therefore important to evaluate your JEA users' extended permissions to see if there are other ways they could gain privileged access to the system.
+One of the core principles of JEA is that it allows non-admins to do *some* admin tasks. JEA doesn't
+protect against users who already have administrator privileges. Users who belong "domain admins",
```

Comment:
remove quotes around domain admins, local admins, and just-in-time

Reply:
fixed